zerver/lib/zulip_update_announcements.py

```diff
@@ -0,0 +1,177 @@
+import os
+import re
+from datetime import timedelta
+from typing import List, Optional
+
+from django.conf import settings
+from django.db import transaction
+from django.db.models import Q, QuerySet
+from django.utils.timezone import now as timezone_now
+
+from zerver.actions.message_send import (
+    do_send_messages,
+    internal_prep_huddle_message,
+    internal_prep_stream_message,
+)
+from zerver.lib.message import SendMessageRequest
+from zerver.models.realm_audit_logs import RealmAuditLog
+from zerver.models.realms import Realm
+from zerver.models.users import UserProfile, get_system_bot
+
+zulip_update_announcements_file_path = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+    "management/commands/zulip_update_announcements.md",
+)
+
+
+def get_latest_zulip_update_announcements_level() -> int:
+    with open(zulip_update_announcements_file_path) as zulip_updates:
+        content = zulip_updates.read()
+        match = re.search(r"\*\*Zulip update announcements level (\d+)\*\*", content)
+        assert match is not None
+        return int(match.group(1))
```

Comment:
So I think @alya's proposal is that we should just configure them as `"""` sections in this Python file, rather than having a specially formatted Markdown file. The motivation for that is that if we ever want to make some of them conditional on anything (say, organization type), that'll work.